packages/fxa-auth-server/test/scripts/write-emails-to-disk.js

```diff
@@ -18,6 +18,7 @@ const cwd = path.resolve(__dirname, ROOT_DIR);
 const execOptions = {
   cwd,
   env: {
+    ...process.env,
```

Comment:
Random Q: Is it weird that we have this block of code copy-pasted in ~at least~ 5 templates? Should we export L17-27 as some helper function somewhere to make this a bit DRYer?
Also interesting is that this file is the only one that is slightly different, in that it includes `stdio: "ignore",` whereas the other four files don't.

It seems like these options are only used w/ `.execSync()` (see this file L32) or a custom `.execAsync()` function (see packages/fxa-auth-server/test/scripts/verification-reminders.js#L13 above).
```js
const execAsync = util.promisify(cp.exec);
```
&hellip; except sometimes we don't use `execOptions` and we just pass `{ cwd }`:
```sh
git grep -A3 -En "\.execAsync\(" packages
```

Reply:
Well off-topic (and on-brand for me)... Also interesting in my over-grepping, looks like we promisify this `cp.exec` in 9 files in 3 packages (auth-server, customs-server, fxa-shared). Sometimes to a new variable, sometimes doing a risky shim into `child_process` module.

```sh
git grep -n "promisify(cp.exec)" packages | cat

packages/fxa-auth-server/lib/version.js:17:const processExec = promisify(cp.exec);
packages/fxa-auth-server/test/scripts/bulk-mailer.js:19:cp.execAsync = promisify(cp.exec);
packages/fxa-auth-server/test/scripts/delete-account.js:13:const execAsync = util.promisify(cp.exec);
packages/fxa-auth-server/test/scripts/dump-users.js:16:cp.execAsync = promisify(cp.exec);
packages/fxa-auth-server/test/scripts/email-config.js:16:cp.execAsync = promisify(cp.exec);
packages/fxa-auth-server/test/scripts/must-reset.js:16:cp.execAsync = promisify(cp.exec);
packages/fxa-auth-server/test/scripts/verification-reminders.js:13:const execAsync = util.promisify(cp.exec);
packages/fxa-customs-server/test/scripts/block-ip.js:9:cp.execAsync = Promise.promisify(cp.exec);
packages/fxa-shared/test/scripts/feature-flags.js:16:const exec = util.promisify(cp.exec);
```

Reply:
Maybe a follow up should be filed for this. Not a fan of that shim either.